src/features/rewards/panelWelcome/index.tsx

```diff
@@ -79,42 +91,51 @@ export default class PanelWelcome extends React.PureComponent<Props, {}> {
           <StyledDescText>
             {getLocale(this.locale.desc)}
           </StyledDescText>
-            {
-              this.props.creating && !this.props.error
-              ? <StyledJoinButton
-                level='secondary'
+          {
+            this.props.creating && !this.props.error
+            ? <StyledJoinButton
+              level='secondary'
+              size='call-to-action'
+              type='subtle'
+              text={'Creating wallet'}
```

Comment:
please add localisation for this one

Reply:
done, we already have an entry for this one (`braveRewardsCreatingText`), it just wasn't being used.